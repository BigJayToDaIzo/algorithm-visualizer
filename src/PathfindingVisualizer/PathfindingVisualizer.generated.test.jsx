import renderer from 'react-test-renderer';
import React, { Component } from 'react';
import Node from './Node/Node';

import { dijkstra, getNodesInShortestPathOrder } from './algorithms/dijkstra';
import PathfindingVisualizer from './PathfindingVisualizer';

jest.mock('./Node/Node');
jest.mock('./PathfindingVisualizer.css');
jest.mock('./algorithms/dijkstra');

const renderTree = tree => renderer.create(tree);
describe('<PathfindingVisualizer>', () => {
  it('should render component', () => {
    expect(renderTree(<PathfindingVisualizer 
    />).toJSON()).toMatchSnapshot();
  });
  
});