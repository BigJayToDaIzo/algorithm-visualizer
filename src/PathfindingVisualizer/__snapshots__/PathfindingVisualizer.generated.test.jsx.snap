// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PathfindingVisualizer> should render component 1`] = `
Array [
  <button
    onClick={[Function]}
  >
    Visualize Dijkstra's Algorithm
  </button>,
  <div
    className="grid"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>,
]
`;
